### PR TITLE
Add GitHub badges to site

### LINKED ISSUE #172: Add a widget showing github activity for a project on ontology detail page

### LINKED ISSUE #1687: Add "last commit" badges to homepage and ontology page
Shields.io will generate all sorts of nice badges that give summary-level information on repositories that can be formatted in using the following markdown:

```markdown
![GitHub last commit](https://img.shields.io/github/last-commit/:organization/:name)
```

It would be useful to include this on one or both of the homepage and ontology-specific pages to give users directly more insight into how maintained an ontology has been lately, regardless of its status as "active", "orphaned", etc.

A few examples:

| Ontology | Repo | Badge |
|----------|-------|--------|
| PLANA | obophenotype/planaria-ontology | GitHub last commit |
| CHEMINF | semanticchemistry/semanticchemistry | GitHub last commit |
| INO | INO-ontology/ino | GitHub last commit |

Caveats:
- I'm not sure what the color scheme is, but I guess it's an opinionated traffic light scheme for when things are getting stale.
- About 3/4 of ontologies in OBO Foundry have structured annotations for the repository. Almost all of those annotations correspond to GitHub repositories. See comment [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1688#issuecomment-989857410)

Note: I'd be happy to provide the implementation of this if it gets positive feedback.

### LINKED ISSUE #1688: Add GitHub stars badges to homepage and ontology page
Similar to #1687, Shields.io will generate all sorts of nice badges that give summary-level information on repositories that can be formatted in using the following markdown:

```markdown
![GitHub Repo stars](https://img.shields.io/github/stars/:organization/:repository?style=social)
```

GitHub stars in some ways could be considered as a proxy for ontology usage and uptake, which would be useful for potential users/consumers.

A few examples:

| Ontology | Repo | Badge |
|----------|-------|--------|
| PLANA | obophenotype/planaria-ontology | GitHub Repo stars |
| CHEMINF | semanticchemistry/semanticchemistry |  GitHub Repo stars |
| INO | INO-ontology/ino |  GitHub Repo stars |

Note: I'd be happy to provide the implementation of this if it gets positive feedback.

### PR DESCRIPTION
**Pull request by cthoyt**
Closes #1688, closes #1687, closes #172

This PR does the following:

1. Switches twitter badge to use shields.io (it looks nicer) and remove unused twitter code (should have been taken care of in #1691)
2. Add GitHub star badge to homepage
3. Add GitHub stars badge, last commit badge, and contributors badge to ontology detail page
4. Remove the google plus link generator - this service does not exist anymore
5. Adds notes in the development docks for running jekyll locally via docker

## Screenshots

I built the site locally with Jekyll and generated the following screenshots

Homepage:

Screen Shot 2021-12-17 at 09 51 26

Ontology detail page:

Screen Shot 2021-12-17 at 09 50 44

### CODE REVIEW

File: _layouts/ontology_detail.html [hunk 1]
```diff
@@ -17,17 +17,23 @@ <h1>
...
-           data-dnt="true">
-          Follow @{{ page.twitter }}
+        <a href="https://twitter.com/{{ page.twitter }}">
+          <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/{{ page.twitter }}?style=social">
         </a>
-        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
       {% endif %}
       {% if page.google_plus %}
-      <a href="https://plus.google.com/{{page.google_plus}}">G+</a>
+        <a href="https://plus.google.com/{{page.google_plus}}">G+</a>
```
Comment (matentzn) [inline _layouts/ontology_detail.html:36]: Google plus does not exist anymore, I think this can be removed

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
This is _super cool_! Thank you!

As these are some pretty big changes to the UI, lets wait and see what others think, but that's really nice!

File: _layouts/ontology_detail.html (hunk 1 above)
Comment (cthoyt) [reply _layouts/ontology_detail.html]: Done

**Review by cmungall — APPROVED** [review]
awesome!

**Comment by cmungall** [comment]
* Fixes #172 

I love it when old outstanding issues are fixed!

**Comment by cthoyt** [comment]
Thoughts on also making it more prominent on the home page when each site was last updated (this could make it easier for users to figure out which projects are not so active anymore)?

Also, what's the way to move this forwards?

**Comment by cmungall** [comment]
This would be such an improvement, I am all for

anticipating potential objections

 1. some ontologies like CHEBI don't commit to their repo
 2. some ontologies may use bitbucket/gitlab
 3. if the same github repo is shared by >1 ontology this can be confusing

However, I don't see any of these as blocker.

For 1, we could potentially add more metadata that indicates the source of an ontology. This would be super-useful to have anyway. This metadata could drive what is shown. But I think this is super-minor and could be done later

For 2, the consequences are simply that no information would be shown, which I think is completely acceptable. We could later extend the badges to include these repos. AFAIK the only repos to use non-GH are dron and ogg which both use bb.

For 3, this would affect many of the stage ontologies, which all share obophenotype/developmental-stage-ontologies. Arguably it would be better to bud these off into their own repos, but that won't happen overnight.

For some merged ontologies like TAO we have entries like:

```
tracker: https://github.com/obophenotype/uberon/issues?q=is%3Aissue+label%3ATAO
```

IMO this is not so useful and we should remove that

But this is still a minor concern for the stage ontologies. I vote to merge, and then iterate on improvements. E.g. if a repo is shared, show an asterisk next to it with a note explaining this.

**Comment by cthoyt** [comment]
1. For repositories like ChEBI that don't commit to their repositories, this could indeed be misleading. On the other hand, the fact that they don't do their curation in the open causes a ton of pain for many reasons, and this might be the only ontology for which this is the case.
2. I looked into this on the linked issue. Below, I copied the count of which repos are using what kind of trackers (that are annotated in OBO Foundry, at least). See the linked issue https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1688#issuecomment-989857410 for more information on how this was generated.

  | System   |   Count |   Percentage |
|----------|---------:|--------------:|
| GitHub   |     182 |        73.39 |
| None     |      64 |        25.81 |
| Other    |       1 |         0.4  |
| GitLab   |       1 |         0.4  |

3. That's a good point, but can be addressed in a totally different PR.

**Comment by nataled** [comment]
After seeing this implemented, I finally looked up what the stars mean. Turns out they have nothing to do with quality by any measure other than "we use github". Definitely misleading, as most (if not all) users equate stars with quality. 

BTW, CHEBI is not the only ontology that doesn't use github for repos. PRO is another, and there are likely more. Unclear to me what "curation in the open" means in this regard.

**Pull request merged**